<?xml version="1.0" encoding="utf-8"?>

<launch>
  
 <include file="$(find velodyne_pointcloud)/launch/VLP16_points.launch" />
  <!-- Ground segmentation -->
  <node name="ground_segmentation" pkg="linefit_ground_segmentation_ros" type="ground_segmentation_node" output="screen">
    <rosparam command="load" file="$(find linefit_ground_segmentation_ros)/launch/segmentation_params.yaml"/>
    <param name="input_topic" value="/dynamic_point_cloud" />
    <param name="ground_output_topic" value="ground_cloud" />
    <param name="obstacle_output_topic" value="obstacle_cloud" />
  </node>
    
     <include file="$(find lidar_package)/launch/rviz.launch" />
</launch>
